gov/agendas/deployments.go

```diff
@@ -1,142 +1,115 @@
-// Copyright (c) 2018, The Decred developers
+// Copyright (c) 2018-2019, The Decred developers
 // Copyright (c) 2017, Jonathan Chappelow
 // See LICENSE for details.
 
-package agendadb
+// agendas package manages the various deployment agendas that are directly voted
+// upon with the vote bits in vote transactions.
+
```

Comment:
For godoc to work properly with the package docs, I believe there cannot be a blank line between the comments and `package agendas`: https://blog.golang.org/godoc-documenting-go-code